Create ui-c8y-1023-68-0-support-dtm-asset-and-definition-api.md (#4528)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-68-0-support-dtm-asset-and-definition-api.md

---
date: ""

> title: Digital Twin Manager asset and definition API support in Asset properties selector

Reviewer: ```suggestion
title: Digital Twin Manager Asset and Definition API support in asset properties selector
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_

>     label: Web SDK

Reviewer: Component should be Digital Twin Manager, I suppose. How about the build artifact? Is ui-c8y correct?

Reviewer 2: We are using DTM API but in our Web SDK (asset properties selector is part of Web SDK). so label and component are fine.

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64674
version: 1023.68.0
---

> Digital Twin Manager enables schema-based modeling in Cumulocity by allowing you to create asset definitions that define the structure and constraints for asset instances. The Asset API lets you manage asset instances based on these predefined asset definitions. When you use the asset properties selector, the system now requests the relevant definition for any asset created with Digital Twin Manager and displays the properties from that definition. This ensures that asset instances created from an asset model inherit and display the correct structure and constraints defined in the model. For more information about the Digital Twin Manager API, refer to the Cumulocity API documentation.

Reviewer: ```suggestion
Digital Twin Manager enables schema-based modeling in {{< product-c8y-iot >}} by allowing you to create asset definitions that define the structure and constraints for asset instances. The Asset API lets you manage asset instances based on these predefined asset definitions. When you use the asset properties selector, the system now requests the relevant definition for any asset created with Digital Twin Manager and displays the properties from that definition. This ensures that asset instances created from an asset model inherit and display the correct structure and constraints defined in the model. For more information about the Digital Twin Manager API, refer to the [{{< openapi >}}](https://cumulocity.com/api/).
```

---
date: ""

> title: Support DTM Asset and Definition API

Reviewer 2: ```suggestion
title: Digital Twin Manager asset and definition API support in Asset properties selector
```
